vmaas_sync/pkg_sync.go

```diff
@@ -101,8 +101,27 @@ func storePackageStrings(tx *gorm.DB, data map[utils.Nevra]vmaas.PackagesRespons
 
 func storePackageDetails(tx *gorm.DB, advisoryIDs map[utils.Nevra]int, nameIDs map[string]int,
 	data map[utils.Nevra]vmaas.PackagesResponsePackageList) error {
-	toStore := make([]models.Package, 0, len(data))
+	inserts := make([]models.Package, 0, len(data))
+	updates := make([]models.Package, 0, len(data))
 
+	names := make([]string, 0, len(data))
+	evras := make([]string, 0, len(data))
```

Comment:
Is there any check whether data set is not empty?

Reply:
There isn't one now, I think it shouldn't cause any problems.